src/services/version-control-system-integration-services/list-repositories-service.js

```diff
@@ -35,7 +37,10 @@ const parseHttpResponse = (httpResponse) => {
       return httpResponse.body
     case 400:
       let errorMessage = httpResponse.body.error
-      errorMessage = errorMessage === 'Invalid scope.' ? 'Please, try another Scope.' : errorMessage
+      errorMessage =
+        errorMessage === 'Invalid scope.'
+          ? 'Your scope does not contain any valid repositories.'
```

Comment:
Great! With this, we could probably archive this issue: https://aziontech.atlassian.net/browse/UXE-2826